src/commands/build-application.ts

```diff
@@ -97,7 +98,13 @@ function replaceBuildPath(filePath) {
         if (err) {
             return console.log(err);
         }
+        if(!vars._isWindows){
```

Comment:
Why is this required ? This will impact build application for initialize VS / normal Vscode scenarios.